### PR TITLE
Student view feedback results: not show self responses.

### LINKED ISSUE #6290: Student: view session results: do not show self responses
Current:
When students view session results, they can see responses they gave to others. 

Suggestion:
Don't show responses the user gave to others. Instead, give a link to show the corresponding submission page. 

Justification:
Self-responses clutter the page. Some students get confused by self-responses.

### PR DESCRIPTION
**Pull request by hackosinn**
Fixes #6290

### CODE REVIEW

**Comment by junhaoyap** [comment]
@hackosinn this is not the right fix, at TEAMMATES we use the HTML files for testing against, note that the tests have failed on our Continuous Integration (CI) platform and that is because the HTML generated from our .tag files do not match what you have just edited

This issue might be more complex so you might want to find d.firstTimer issues in the meantime before revisiting this in the near future

Feel free to continue on this if you want some challenge, you can always ping us for help whenever necessary

Note that the CI showing a green (pass) is necessary for us to consider it for merge

**Comment by hackosinn** [comment]
do i have to modify the .tag files as well? i'm understanding the structure
of the app more clearly now.thanks for help.

**Comment by hackosinn** [comment]
@junhaoyap - can u guide me how should i approach for solution

File: src/test/resources/data/StudentFeedbackResultsPageUiTest.json [hunk 1]
```diff
@@ -425,7 +425,7 @@
       },
       "questionNumber": 5,
       "questionType": "TEXT",
-      "giverType": "STUDENTS",
+-      "giverType": "STUDENTS",
```
Comment (junhaoyap) [inline src/test/resources/data/StudentFeedbackResultsPageUiTest.json]: This is unecessary

**Comment by junhaoyap** [comment]
@hackosinn, what you will want to do in this case is to find out which ```.tag``` file generates the relevant HTML, there is no need to edit the HTML manually in TEAMMATES as this is taken care of automatically, cc-ing @wkurniawan07 to explain the details of the automagic change in HTML

As to how one would find the relevant tag file, you can start by searching for ```.tag``` files with "feedback" as part of the name to narrow down your searches, being able to understand the templating system and how each file relates to each will take some effort. In order to do that, look at the name of the ```.tag``` file you have chosen and search for where it is being used, with some effort and time and reading, you'll be able to understand which file uses which file for templating

**Pull request closed without merging**